assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -173,7 +176,24 @@ const MediaElement = ({
 
   const onClick = () => onInsert(resource, width, height);
 
+  /// Callback for when the More dropdown menu has been opened / closed.
+  const menuCallback = (isMenuOpen) => {
+    if (isMenuOpen && type === 'video' && mediaElement.current) {
+      // If it's a video, pause the preview while the dropdown menu is open.
+      mediaElement.current.pause();
+      mediaElement.current.currentTime = 0;
+    } else if (!isMenuOpen) {
+      setShowMoreIcon(false);
```

Comment:
This causes odd behavior:

1. Click on icon
1. Menu opens
1. Video stops playing
1. Click on icon again
1. Menu closes
1. Icon disappears
1. Video still not playing

What I'd expect:

1. Click on icon
1. Menu opens
1. Video stops playing
1. Click on icon again
1. Menu closes
1. Video continues playing

It gets weirder:

1. Click on icon
1. Menu opens
1. Video stops playing
1. Move cursor away from video
1. `currentTime` is reset
1. Move cursor over video again
1. Video continues playing, with menu open

I'd say the video should never continue playing as long as the menu is open.

Reply:
Redid this part to use `useEffect` instead.